examples/LoRandom/LoRandom.ino

```diff
@@ -0,0 +1,92 @@
+/*
+  LoRandomSeed
+
+  Example demonstrating how to generate random numbers using LoRa.
+
+  created 7 Sep 2020
+  by Kongduino
+  adapted by Andres Sabas
+  
+  SX1276 Register (address)     Register bit field (bit #)      Values    Note
```

Comment:
I would suggest to remove the register notes from the example, since they are not used in the example.